src/html/why.html

```diff
@@ -0,0 +1,52 @@
+@@include("partials/head.html", {"tagline": false})
+
+    <main>
+        <div class="container pt3 pb3">
+
+            <h2>Why Meadowrun?</h2>
+
+            <p>The short answer is: Meadowrun runs your Python code in the cloud, seamlessly. Scale up to a bigger machine, or scale out to many machines – once you are ready to run your analysis or program on a dataset that doesn’t fit on your laptop, Meadowrun helps you do that without hassle and while keeping many of the benefits of running locally.</p>
+
+            <br>
+
+            <h3>What’s the Problem?</h3>
+
+            <p>That all sounds great, I hear you say, but don’t we already have containers, AWS/Azure/GCP, Kubernetes and Dask/RAPIDS/Spark? Indeed we do, but putting those together while ensuring an efficient development workflow is no walk in the park.</p>
+
+            <p>For example, containers are very useful for running on scalable compute infrastructure like Kubernetes, but locally you’re more likely to have a virtual environment with third-party packages installed, and you’ll certainly have some local code you’d like to run. How do you get that in the container image efficiently and reproducibly? And if you do get a container running, how do you see the logs? If there are any results how are they sent back?</p>
+
+            <p>Another source of complexity is the wide array of options to choose from, even within a single cloud provider like AWS. Should you allocate an EC2 instance, and if so, should it be a on-demand instance or a spot instance? Can you use Lambdas, or maybe Elastic Container Service? All of these have more options to choose from each with different costs and functional characteristics.</p>
```

Comment:
I would maybe just get rid of this last part: ~All of these have more options to choose from each with different costs and functional characteristics.~

I think you're trying to say that even if you decide you're using e.g. ECS, there's still tons more configuration to deal with within that option?